spi/common/core-spi/src/main/java/org/eclipse/edc/spi/monitor/ConsoleMonitor.java

```diff
@@ -76,16 +84,35 @@ public void debug(Supplier<String> supplier, Throwable... errors) {
 
     private void output(String level, Supplier<String> supplier, Throwable... errors) {
         var time = ZonedDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
-        System.out.println(prefix + level + " " + time + " " + sanitizeMessage(supplier));
+        var colorCode = getColorCode(level);
```

Comment:
the `useColor` evaluation could be done here instead of in the `getColorCode` method:
```
        var colorCode = useColor ? getColorCode(level) : "";
        var resetCode = useColor ? ConsoleColor.RESET : "";
```